pypiserver/manage.py

```diff
@@ -171,11 +169,11 @@ def update(pkgset, destdir=None, dry_run=False, stable_only=True):
 def update_all_packages(
     roots, destdir=None, dry_run=False, stable_only=True, blacklist_file=None
 ):
-    all_packages = itertools.chain(*[core.listdir(r) for r in roots])
+    all_packages = itertools.chain(*[listdir(r) for r in roots])
```

Comment:
I know you didn't write the original, but since we're touching it, we can keep this (potentially) lazy by updating like:

```suggestion
    all_packages = itertools.chain.from_iterable(listdir(r) for r in roots)
```

Reply:
Sure, I wanted to touch manage.py as little as possible, but this totally makes sense